test/testsuite/lantest.c

```diff
@@ -1251,18 +1733,17 @@ void usage(char *av0)
 }
 
 /* ------------------------------- */
-int main(int ac, char **av)
+int main(int32_t ac, char **av)
 {
-    int cc, i, t;
-    int Debug = 0;
+    int32_t cc, i, t;
     static char *vers = "AFP3.4";
     static char *uam = "Cleartxt Passwrd";
 
     if (ac == 1) {
         usage(av[0]);
     }
 
-    while ((cc = getopt(ac, av, "1234567bCGgVvF:f:h:n:p:s:u:w:")) != EOF) {
+    while ((cc = getopt(ac, av, "1234567bcGgKVvF:f:h:n:p:s:u:w:")) != EOF) {
```

Comment:
I spotted a preexisting bug here: The optarg string checks for `1` and `2` but the following switch statement doesn't cover those cases.

The lantest cannot handle AFP 2.1 or 2.2 the last time I tested, but crashes after the first test. I haven't tested it after your changes though.

Reply:
Done.